modules/junit-jupiter/src/test/java/org/testcontainers/junit/jupiter/TestLifecycleAwareContainerMock.java

```diff
@@ -0,0 +1,35 @@
+package org.testcontainers.junit.jupiter;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.lifecycle.TestDescription;
+import org.testcontainers.lifecycle.TestLifecycleAware;
+
+import java.util.Optional;
+
+public class TestLifecycleAwareContainerMock extends GenericContainer implements TestLifecycleAware {
+
+    private int numBeforeTestsCalls = 0;
+    private int numAfterTestsCalls = 0;
+
+    public TestLifecycleAwareContainerMock() {
+        super("httpd:2.4-alpine");
```

Comment:
Can we use a lighter container here, for marginally quicker creation and start?
e.g. https://github.com/testcontainers/testcontainers-java/blob/c91513d9ebfb3051c2ddaf9b0581e7dbb10e236a/core/src/test/java/org/testcontainers/junit/FileOperationsTest.java#L24-L25